src/dist/replication/lib/replica_backup.cpp

```diff
@@ -421,6 +418,15 @@ static bool backup_parse_dir_name(const char *name,
     }
 }
 
+void replica::background_clear_backup_checkpoint(const std::string &policy_name)
+{
+    tasking::enqueue(LPC_BACKGROUND_COLD_BACKUP,
+                     &_tracker,
+                     [this, policy_name]() { clear_backup_checkpoint(policy_name); },
+                     get_gpid().thread_hash(),
+                     std::chrono::minutes(options()->cold_backup_checkpoint_reserve_minutes));
```

Comment:
We don't restrict backup interval time, which is defined by user who created backup policy, it will be safety to check `cold_backup_checkpoint_reserve_minutes < policy.interval_minute`. 
`clear_backup_checkpoint` will list all checkpoints created by the specific policy, if policy interval_minute is shorter than cold_backup_checkpoint_reserve_minutes, it is possible to remove checkpoints which may be useful during backup.

Reply:
when cold backup is completed, primary replica will send backup_request,  which backup_id = 0, to secondaries. So when the secondaries receive this backup_request, they can't get nothing about this cold backup policy. What they can do are simpely remove the backup dir.
Maybe we can refactor these code firstly.

Reply:
Good point. I suggest adding a debug log here for this edge case.